internal/wrappers/projects-http.go

```diff
@@ -148,6 +147,32 @@ func (p *ProjectsHTTPWrapper) GetByID(projectID string) (
 	return handleProjectResponseWithBody(resp, err, http.StatusOK)
 }
 
+func (p *ProjectsHTTPWrapper) GetByName(name string) (
+	*ProjectResponseModel,
+	*ErrorModel,
+	error) {
+	params := make(map[string]string)
+	params["name"] = name
+	resp, _, err := p.Get(params)
+	if err != nil {
+		return nil, nil, err
+	}
+	setLimit(params, "10")
```

Comment:
can be removed, it exists in the get call

Reply:
done